script/Deploy.s.sol

```diff
@@ -70,24 +79,24 @@ abstract contract Deploy is Common, Script {
     if (_chainId == 31_337) {
       mintAirdrop();
     }
-
```

Comment:
Do you have some linter on different from ours?

Reply:
I'm running the project linter, but I tend to break the lines or group line for readability